panel/template/base.py

```diff
@@ -5,6 +5,7 @@
 from __future__ import absolute_import, division, unicode_literals
 
 import os
+from panel.layout.grid import GridSpec
```

Comment:
Imports should be relative and be ordered correctly, i.e. you should use `from ..layout.grid import GridSpec` and then move this import into the list of internal imports.